### ISSUE TITLE
Problem with Robloach Component INstaller

### ISSUE THREAD

**Issue by sneps85** [TYPO3/CmsComposerInstallers#11]
If a TYPO3-Extension depends on a Robloach-Component (e.g. eonasdan bootstrap-datetime-picker, moment, ..) the extension will be installed on the wrong place (in Packages/Libraries instead of typo3conf/ext). Could the issue lay in your component installer?

**Comment by Cash2m** [comment]
No, this is [because TYPO3 tells so](https://github.com/TYPO3/TYPO3.CMS/blob/TYPO3_6-2-12/composer.json#L27). But if you are on 6.x branch there is still no full composer support. There will be a TYPO3 7.x update next week with focus on composer support (so says the roadmap).

**Comment by tmaroschik** [comment]
Hey @sneps85, during our April Sprint we identified some blockers for full composer support. It won't make it into 7.3 but probably into one of the versions until LTS. We have to change typo3 a bit more than we anticipated for that. But things will become leaner and faster.

Regarding your issue, has your extension the correct "type" set up in the extensions composer.json? It should be 'typo3-cms-extension'. Otherwise the installer doesn't identify it as an extension.
